types/react-select/src/components/MultiValue.d.ts

```diff
@@ -34,6 +34,7 @@ export const MultiValueContainer: typeof MultiValueGeneric;
 export const MultiValueLabel: typeof MultiValueGeneric;
 export type MultiValueRemoveProps<OptionType extends OptionTypeBase> = CommonProps<OptionType, true> & {
   children: ReactNode,
+  data: OptionType,
```

Comment:
Looks good (as do the other props)

https://github.com/JedWatson/react-select/blob/v3.0.0/packages/react-select/src/components/MultiValue.js#L48

I notice we use `data: OptionType` throughout (while `data: any` seems to be used in react-select)

I assume that's okay and just additional typesafety for us 👍

Reply:
Yup.